lib/Server.js

```diff
@@ -384,8 +387,8 @@ class Server {
       });
     } else {
       // route content request
-      this.app.get(
-        '*',
+      this.app.use(
+        this.options.contentBasePrependPublic ? this.options.publicPath : '/',
```

Comment:
Breaking change

Reply:
@evilebottnawi actually this is not a breaking change `express.static` will handle only `GET` requests, any other will be ignored